spring-boot/pom.xml

```diff
@@ -44,7 +44,7 @@
 		<dependency>
 			<groupId>org.json</groupId>
 			<artifactId>json</artifactId>
-			<version>20210307</version>
+			<version>20220924</version>
```

Comment:
did this change on its own when you updated sentry-spring-boot and sentry-logback?

or was this part of an Upgrade Pathway per the sentry sdk instructions?

@Angelodaniel

Reply:
This was due to a command for Maven which retrieved the latest packages.
When upgrading the dependencies these got upgraded as well 
https://stackoverflow.com/questions/27798691/maven-force-update-only-for-specific-dependency-mvn-u-for-selected-dependencie

Reply:
ok great we'll go with it